### PR TITLE
Update nginx in kvm

### PR DESCRIPTION
**Pull request by cokiengchiara**
Towards https://github.com/giantswarm/giantswarm/issues/15428

Requesting Team Rocket for review on KVM.

### CODE REVIEW

**Comment by ubergesundheit** [comment]
Re: https://github.com/giantswarm/giantswarm/issues/15428#issuecomment-769212883

What about linking this List of "Preinstalled and optional Apps" here?

File: src/content/guides/installing-optional-ingress-controller/index.md [hunk 1]
```diff
@@ -14,13 +14,10 @@ An ingress controller helps you expose your services to the outside world.
 
 ## Which workload cluster releases do not ship with an ingress controller
 
-Currently clusters on our AWS and Azure installations that have a workload cluster release version
-newer than `10.0.0` (AWS) or `12.0.0` (Azure) ship without an ingress controller by default. That
-allows you full control to choose which and how many ingress controllers you
-want to run on your cluster.
+Clusters on our AWS, Azure, and KVM (On-premises) installations that have a workload cluster release version newer than `10.0.0` (AWS), `12.0.0` (Azure), and `12.2.0` (KVM) ship without an ingress controller by default.
```
Comment (cokiengchiara) [inline src/content/guides/installing-optional-ingress-controller/index.md:17]:
```suggestion
Clusters on our AWS, Azure, and KVM (On-premises) installations that have a workload cluster release version newer than `10.0.0` (AWS), `12.0.0` (Azure), and `12.2.0` (KVM) ship without an ingress controller by default. (Related: [Preinstalled and optional apps in workload clusters](https://docs.giantswarm.io/reference/workload-cluster-release-versions/#apps))
```

Comment (cokiengchiara) [reply src/content/guides/installing-optional-ingress-controller/index.md:17]: @ubergesundheit what's the right relative link 🙈

Comment (ubergesundheit) [reply src/content/guides/installing-optional-ingress-controller/index.md:17]:
```suggestion
Clusters on our AWS, Azure, and KVM (On-premises) installations that have a workload cluster release version newer than `10.0.0` (AWS), `12.0.0` (Azure), and `12.2.0` (KVM) ship without an ingress controller by default. (Related: [Preinstalled and optional apps in workload clusters]({{< relref "/content/reference/workload-cluster-release-versions/_index.md#apps" >}}))
```

**Review by ubergesundheit — APPROVED** [review]
Bueno

File: src/content/guides/installing-optional-ingress-controller/index.md [hunk 2]
```diff
@@ -18,13 +18,10 @@ An ingress controller helps you expose your services to the outside world.
 
 ## Which workload cluster releases do not ship with an ingress controller
 
-Currently clusters on our AWS and Azure installations that have a workload cluster release version
-newer than `10.0.0` (AWS) or `12.0.0` (Azure) ship without an ingress controller by default. That
-allows you full control to choose which and how many ingress controllers you
-want to run on your cluster.
+Clusters on our AWS, Azure, and KVM (On-premises) installations that have a workload cluster release version newer than `10.0.0` (AWS), `12.0.0` (Azure), and `12.2.0` (KVM) ship without an ingress controller by default. (Related: [Preinstalled and optional apps in workload clusters]({{< relref "/content/reference/workload-cluster-release-versions/_index.md#apps" >}}))
```
Comment (marians) [inline src/content/guides/installing-optional-ingress-controller/index.md:21]: BTW, no need to uppercase `On-premises`

Comment (cokiengchiara) [reply src/content/guides/installing-optional-ingress-controller/index.md:21]: oh i missed this comment haha noted.

**Pull request merged**